packages/nimbus/src/patterns/fields/date-range-picker-field/date-range-picker-field.dev.mdx

```diff
@@ -0,0 +1,478 @@
+```
+
+### Basic usage
+
+The simplest implementation with a label and controlled state:
+
+```jsx-live-dev
+const App = () => {
+  const [value, setValue] = useState<DateRange | null>(null);
+
+  return (
+    <DateRangePickerField
+      label="Select Date Range"
+      value={value}
+      onChange={setValue}
+    />
+  );
+}
+```
+
+### Working with date values
+
+DateRangePickerField uses the same `@internationalized/date` library as DateRangePicker.
+
+## Usage examples
+
+### With description
+
+Add helper text below the input to provide context:
+
+```jsx-live-dev
+const App = () => {
+  const [value, setValue] = useState<DateRange | null>(null);
+
+  return (
+    <DateRangePickerField
+      label="Event Duration"
+      description="Select the start and end dates for your event"
+      value={value}
+      onChange={setValue}
+    />
+  );
+}
+```
+
+### With validation errors
+
+Show error messages when the field is touched and has validation errors:
+
+```jsx-live-dev
+const App = () => {
+  const [value, setValue] = useState<DateRange | null>(null);
+  const [touched, setTouched] = useState<boolean>(false);
+
+  // Simulate validation errors
+  const errors = !value && touched ? { missing: true } : undefined;
+
+  return (
+    <DateRangePickerField
+      label="Required Date Range"
+      description="Please select a date range"
+      value={value}
+      onChange={setValue}
+      onBlur={() => setTouched(true)}
+      errors={errors}
+      touched={touched}
+      isRequired
+    />
+  );
+}
+```
+
+**Note**: Errors only display when both `touched={true}` and `errors` exist. This prevents showing errors before the user interacts with the field.
+
+### With info popover
+
+Provide additional context through an info button and popover:
+
+```jsx-live-dev
+const App = () => {
+  const [value, setValue] = useState<DateRange | null>(null);
+
+  return (
+    <DateRangePickerField
+      label="Report Period"
+      description="Select the date range for your report"
+      info="Date ranges are inclusive. Both the start and end dates will be included in the report results."
+      value={value}
+      onChange={setValue}
+    />
+  );
+}
+```
+
+### Size variants
+
+Two size options are available to match your form density:
+
+```jsx-live-dev
+const App = () => {
+  const [valueSm, setValueSm] = useState<DateRange | null>(null);
+  const [valueMd, setValueMd] = useState<DateRange | null>(null);
+
+  return (
+    <Stack direction="column" gap="400">
+      <DateRangePickerField
+        label="Small Size"
+        description="Compact layout for dense forms"
+        size="sm"
+        value={valueSm}
+        onChange={setValueSm}
+      />
+      <DateRangePickerField
+        label="Medium Size"
+        description="Standard size for most forms"
+        size="md"
+        value={valueMd}
+        onChange={setValueMd}
+      />
+    </Stack>
+  );
+}
+```
```

Comment:
oh, well look at that. this is kind of what i was suggesting [here](https://github.com/commercetools/nimbus/pull/646/files#r2492089464)